Sources/ScipioKit/Producer/PIF/FrameworkBundleAssembler.swift

```diff
@@ -155,9 +147,66 @@ struct FrameworkBundleAssembler {
     }
 
     private func copyResources() throws {
-        if let resourceBundlePath = frameworkComponents.resourceBundlePath {
-            let destinationPath = frameworkBundlePath.appending(component: resourceBundlePath.basename)
-            try fileSystem.copy(from: resourceBundlePath, to: destinationPath)
+        func copyInfoPlist() throws {
+            let sourcePath = frameworkComponents.infoPlistPath
+            let destinationPath = frameworkBundlePath.appending(component: "Info.plist")
+            try fileSystem.copy(from: sourcePath, to: destinationPath)
+        }
+
+        /// Retruns the resulting, copied resource bundle path.
+        func copyResourceBundle() throws -> TSCAbsolutePath? {
+            if let sourcePath = frameworkComponents.resourceBundlePath {
+                let destinationPath = frameworkBundlePath.appending(component: sourcePath.basename)
+                try fileSystem.copy(from: sourcePath, to: destinationPath)
+                return destinationPath
+            } else {
+                return nil
+            }
+        }
+
+        // Move PrivacyInfo.xcprivacy to expected location (if exists in the resource bundle)
+        //
+        // ref: https://developer.apple.com/documentation/bundleresources/adding-a-privacy-manifest-to-your-app-or-third-party-sdk#Add-a-privacy-manifest-to-your-framework
+        func movePrivacyInfoIfNeeded(
+            resourceBundlePath: TSCAbsolutePath,
+            relativePrivacyInfoPath: TSCRelativePath
+        ) throws {
+            let privacyInfoPath = resourceBundlePath.appending(relativePrivacyInfoPath)
+            if fileSystem.exists(privacyInfoPath) {
+                try fileSystem.move(
+                    from: privacyInfoPath,
+                    to: resourceBundlePath.parentDirectory.appending(component: relativePrivacyInfoPath.basename)
+                )
+            }
+        }
```

Comment:
As `copyInfoPlist`, how about appending "PrivacyInfo.xcprivacy" by this function. There is not big difference with normal move function with this.

Reply:
I think PrivacyInfo will not be accessed from user code (usually), so instead of duplicating it in a framework and its resource bundle, extracting it is better. Duplicating the file increases the framework size a little bit.